crates/rpc/rpc/src/debug.rs

```diff
@@ -388,7 +396,7 @@ where
                                 Ok(frame.into())
                             })
                             .await?;
-                        return Ok(frame)
+                        return Ok(frame);
```

Comment:
```suggestion
                        return Ok(frame)
```
should also configure the editor to not insert semicolons

Reply:
Done with `make fmt`